bin/pre-release.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+ALL_PACKAGE_VERSION=$(node -p "require('./packages/all/package.json').version")
+TAG="v$ALL_PACKAGE_VERSION"
+
+npm run build:packages && npm run build:dist && lerna publish --skip-git --skip-npm && git add . && git commit -m "Release $TAG"
+echo "🗒 All done. Ready to create a pull request. Once approved, run npm run release"
```

Comment:
This message show us even when we have an error or we cancel Lerna. I'll give it a thought, maybe we'll be able to wrap it somehow.
<img width="873" alt="screen shot 2018-02-26 at 12 55 11" src="https://user-images.githubusercontent.com/788096/36672702-312d24d0-1af8-11e8-8502-d5bc6c146daf.png">

Reply:
ok, let's have chat how to best deal with it

Reply:
@alex-ju have pushed an update to prerelease script. how about now?

Reply:
if you cancel lerna publish, exit code is 0

Reply:
yep, looks good now.